README.md

```diff
@@ -58,9 +58,12 @@ As with `seq` and `Seq`, this library comes with a bunch of well-known collectio
 
 ### `taskSeq` computation expressions
 
-The `taskSeq` computation expression can be used just like using `seq`. On top of that, it adds support for working with tasks through `let!` and 
-looping over a normal or asynchronous sequence (one that implements `IAsyncEnumerable<'T>'`). You can use `yield!` and `yield` and there's support
-for `use` and `use!`, `try-with` and `try-finally` and `while` loops within the task sequence expression:
+The `taskSeq` computation expression can be used just like using `seq`.
+Additionally, it adds support for working with `Task`s through `let!` and 
+looping over both normal and asynchronous sequences (ones that implement
+`IAsyncEnumerable<'T>'`). You can use `yield!` and `yield` and there's support
```

Comment:
Damn, there's still too many "and"s here! Probably should be two sentences (also maybe the fact implicit `yield` works should be mentioned?)

Reply:
@bartelink I want to rework that whole section, maybe turn it into a feature table, or link to docs for some part. It could certainly be improved.

Reply:
Yes, there are far worse paragraphs on the planet - at the end of the day it does convey the bore point perfectly well, lets 🙈